virtcontainers/mount.go

```diff
@@ -351,7 +355,7 @@ func bindUnmountAllRootfs(ctx context.Context, sharedDir string, sandbox *Sandbo
 		if c.state.Fstype == "" {
```

Comment:
This check here should prevent `bindUnmountContainerRootfs` from being called. If it were being called, the `bindUnmountContainerRootfs` call would have returned an error which would have been propogated up the stack.
Are you seeing otherwise?

Reply:
@amshinde thanks for your review. I think we still need this change. **bindUnmountContainerRootfs** is called in several places ，there is no check in **https://github.com/kata-containers/runtime/blob/master/virtcontainers/container.go#L1123**. 
in addition，call **bindUnmountContainerRootfs** don't always return  error, see **https://github.com/kata-containers/runtime/blob/master/virtcontainers/mount.go#L329**.

Reply:
Ok fair enough.